File: app/src/main/java/me/larikraun/tourreviews/TourReviewApp.java
package me.larikraun.tourreviews;

import android.app.Application;

import me.larikraun.tourreviews.dagger.AppMainComponent;
import me.larikraun.tourreviews.dagger.ContextModule;
import me.larikraun.tourreviews.dagger.DaggerAppMainComponent;
import me.larikraun.tourreviews.dagger.RepositoryModule;

/**
 * Author: Omolara Adejuwon
 * Date: 17/10/2018.
 */
public class TourReviewApp extends Application {
	//AppMainComponent mComponent;
	
	@Override
	public void onCreate () {
		super.onCreate ();
//		mComponent = DaggerAppMainComponent.builder ()
//				.contextModule (new ContextModule (this))
//				.repositoryModule (new RepositoryModule ())
//				.build ();
	}
}

File: app/src/main/java/me/larikraun/tourreviews/TourReviewApp.kt
package me.larikraun.tourreviews;

import android.app.Application
import uk.co.chrisjenx.calligraphy.CalligraphyConfig

/**
 * Author: Omolara Adejuwon
 * Date: 17/10/2018.
 */
class TourReviewApp : Application() {

	override fun onCreate() {
		super.onCreate ()
		CalligraphyConfig.initDefault(CalligraphyConfig.Builder()
				.setDefaultFontPath("fonts/montserrat/Montserrat-Regular.otf")
				.setFontAttrId(R.attr.fontPath)
				.build()
		)
	}
}
